### DISCUSSION TITLE
大佬好，请问这个网站有api的调用接口吗？比如说按人、按番号搜索之类的

### DISCUSSION THREAD

**Discussion by XC-Li** [libredmm/librefanza#55] [Q&A] [answered]
大佬好，偶尔发现这个很nb的网站，请问这个网站有api的调用接口吗？比如说按人、按番号搜索之类的

**Comment by libredmm** [comment, marked as answer]
搜索单个番号的 JSON API：
https://www.libredmm.com/search.json?q=SDDE-123

按演员搜索的接口暂时没有
